features/admin_associates_customer_with_auction.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Admin associates customer with auction
+  As an administrator
+  I want to be able to associate auctions with customers
+  So that users and admins can see who is creating certain projects
+
+  Scenario: Associating an auction with a customer
+    Given I am an administrator
+    And I sign in
+    And there is an open auction
+    And there is a customer
+    And I visit the auctions admin page
+    
+    When I click to edit the auction
+    Then I should see a select box with all the customers in the system
+    
+    When I select a customer on the form
+    And I click on the "Update" button
+    Then I expect the customer to have been saved
+
+    When I click to edit the auction
+    Then I should see the customer selected for the auction
+
+  Scenario: Seeing the customer on the auction page
+    Given I am an administrator
+    And I sign in
+    And there is an auction with an associated customer
+    When I visit the auction page
+    Then I should see the customer name on the page
+
+    When I visit the admin auction page for that auction
+    Then I should see the customer name on the page
+
+  Scenario: Vendor sees the customer
```

Comment:
seems like these vendor-specific tests shouldn't require signing in-- should we remove user creation / sign in steps to prevent confusion about whether only certain types of ppl can see this info?